### ISSUE TITLE
Version 5.0 - Constant Update Available?

### ISSUE THREAD

**Issue by graciecorgi** [hadley/scales#98]
Hello:

R constantly identifies Scales as having update to 5.0 available - even after updating. I've removed and reinstalled the package, get the update available as it shows I have version 0.4.1, update appears succesfull, but upon restart it still shows 5.0 is available and that I'm on 0.4.1. Any ideas?

**Comment by graciecorgi** [comment]
Sorry, I think I found my problem....scales 5.0 is not available for R 3.4.1. Is that true? I tried installing from a URL this time instead of install.packages("scales"). Here's the detail;


> install.packages("scales")

  There is a binary version available but the source version is later:
       binary source needs_compilation
scales  0.4.1  0.5.0              TRUE

  Binaries will be installed
trying URL 'https://cran.rstudio.com/bin/windows/contrib/3.4/scales_0.4.1.zip'
Content type 'application/zip' length 715665 bytes (698 KB)
downloaded 698 KB

package ‘scales’ successfully unpacked and MD5 sums checked

The downloaded binary packages are in
	C:\Users\tyler.hahn\AppData\Local\Temp\RtmpGcyHqC\downloaded_packages
> help(package="scales")
> remove.packages("scales")
Removing package from ‘C:/Program Files/R/R-3.4.1/library’
(as ‘lib’ is unspecified)
> install.packages("https://cran.r-project.org/src/contrib/scales_0.5.0.tar.gz")
Warning in install.packages :
  package ‘https://cran.r-project.org/src/contrib/scales_0.5.0.tar.gz’ is not available (for R version 3.4.1)
